projects/isy-angular-widgets/src/test.ts

```diff
@@ -4,8 +4,11 @@ import 'zone.js';
 import 'zone.js/testing';
 import {getTestBed} from '@angular/core/testing';
 import {BrowserDynamicTestingModule, platformBrowserDynamicTesting} from '@angular/platform-browser-dynamic/testing';
+import {ɵSharedStylesHost} from '@angular/platform-browser';
 
 // Initialize the Angular testing environment.
 getTestBed().initTestEnvironment(BrowserDynamicTestingModule, platformBrowserDynamicTesting(), {
   teardown: {destroyAfterEach: true}
 });
+
+afterEach(() => getTestBed().inject(ɵSharedStylesHost).ngOnDestroy());
```

Comment:
Why is this expressen needed? (Please comment)

Reply:
Done